structs.go

```diff
@@ -24,3 +24,137 @@ type TagResult struct {
 		} `json:"items"`
 	} `json:"data"`
 }
+
```

Comment:
Why do you have a massive struct of unused data. Fix this and stop copy pasting from JSON to Go.

Reply:
Removed unnecessary data just for you :)